CRM/Report/Form.php

```diff
@@ -3531,17 +3531,17 @@ public function filterStat(&$statistics) {
                     $val[$key] = $options[$valIds];
                   }
                 }
-                $pair[$op] = (count($val) == 1) ? (($op == 'notin' || $op ==
-                  'mnot') ? ts('Is Not') : ts('Is')) : CRM_Utils_Array::value($op, $pair);
+                $pair[$op] = (count($val) == 1) ?
+                  (($op == 'notin' || $op == 'mnot') ? ts('Is Not') : ts('Is')) :
+                  ($pair[$op] ?? '');
```

Comment:
This looks confusing but is mostly whitespace cleanup - the previous line-breaks made no sense.